src/UserManagerSettings.js

```diff
@@ -18,8 +19,10 @@ export default class UserManagerSettings extends OidcClientSettings {
         popupWindowTarget,
         silent_redirect_uri,
         automaticSilentRenew = false,
+        automaticCordovaSilentRenew = false,
```

Comment:
Oops
On 17 Jun 2016 12:21 AM, "Brock Allen" <notifications@github.com> wrote:

> In src/UserManagerSettings.js
> <https://github.com/IdentityModel/oidc-client-js/pull/45#discussion_r67353094>
> :
>
> > @@ -18,8 +19,10 @@ export default class UserManagerSettings extends OidcClientSettings {
> >          popupWindowTarget,
> >          silent_redirect_uri,
> >          automaticSilentRenew = false,
> > +        automaticCordovaSilentRenew = false,
>
> I think this got left in still since the most recent changes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/IdentityModel/oidc-client-js/pull/45/files/37f9ed5191765eed51554225d14bbada615750ee#r67353094>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8zNUi_kwyT4Sh-UNQJu7oewXk9ghnbks5qMVwFgaJpZM4IpDMr>
> .
>